scripts/01b_create_predict_batch.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+PROJECT_PATH="/home/ubuntu/cultionet_train"
+CHUNKSIZE=200
+NUM_WORKERS=6
```

Comment:
oh nice example! 👍 
I see you increase the default chunksize and decrease num_workers

Reply:
These are just example script templates.